docs/pages/database-access/guides/mongodb-atlas.mdx

```diff
@@ -70,13 +70,19 @@ $ teleport db start \
 <TabItem label="Configuration file">
 
 On the node where you will run the Teleport Database Service, run
-`teleport db start` with the following in `/etc/teleport.yaml`:
+`teleport db` with the following in `/etc/teleport.yaml`:
```

Comment:
Is there a difference in how v10 operates? This backport is different from the original or the V11 backport.